jetty-http/src/main/java/org/eclipse/jetty/http/CompressedContentFormat.java

```diff
@@ -0,0 +1,43 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2016 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.http;
+
+/* ------------------------------------------------------------ */
+public class CompressedContentFormat
+{
+    public static final CompressedContentFormat GZIP = new CompressedContentFormat("gzip", ".gz");
+    public static final CompressedContentFormat BR = new CompressedContentFormat("br", ".br");
```

Comment:
@gregw bzip2 is slow decompressing, see https://www.opencpu.org/posts/brotli-benchmarks/